modules/olmv1-installing-an-operator.adoc

```diff
@@ -49,54 +124,113 @@ operator.operators.operatorframework.io/quay-example created
 +
 [source,terminal]
 ----
-$ oc get operator.operators.operatorframework.io/quay-example -o yaml
+$ oc get operator.operators.operatorframework.io pipelines-operator -o yaml
 ----
 +
+[NOTE]
+====
+If you specify a channel or define a version range in your Operator or extension's CR, {olmv1} does not display the resolved version installed on the cluster. Only the version and channel information specified in the CR are displayed.
```

Comment:
Same comment as line 14 about `{olmv1}`. It is already defined in this module, so not sure if there is another common attribute available to you when referencing subsequent instances.